cluster/gce/gci/configure-helper.sh

```diff
@@ -1032,6 +1032,11 @@ EOF
 }
 
 function create-node-problem-detector-kubeconfig {
+  local apiserver_address="${1}"
```

Comment:
Why is this logic being introduced? Can you add some relevant comments?

Reply:
This is to fix https://github.com/kubernetes/node-problem-detector/issues/227. This is consistent with `create-kubelet-kubeconfig`. I think it is the correct and standard way now, so I didn't add any comment.